pkg/sqlcmd/commands.go

```diff
@@ -103,7 +103,7 @@ func newCommands() Commands {
 			isSystem: true,
 		},
 		"ONERROR": {
-			regex:  regexp.MustCompile(`(?im)^[\t ]*?:?ONERROR(?:[ \t]+(.*$)|$)`),
+			regex:  regexp.MustCompile(`(?im)^[\t ]*?:?ON ERROR(?:[ \t]+(.*$)|$)`),

```

Comment:
Does it also handle lowercase?
Should we have one of the unit tests in lowercase variant of this command?

Reply:
The (?im) modifier make the expression case insensitive

See regex101.com for a great interactive tool to play with regex